test/parallel/test-cluster-net-listen-relative-path.js

```diff
@@ -0,0 +1,45 @@
+// Copyright Joyent, Inc. and other Node contributors.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a
+// copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to permit
+// persons to whom the Software is furnished to do so, subject to the
+// following conditions:
+//
+// The above copyright notice and this permission notice shall be included
+// in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+// OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN
+// NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+// OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
+// USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const net = require('net');
+const fs = require('fs');
+
+if (cluster.isMaster) {
+  // ensure that the worker exits peacefully
+  process.chdir(common.tmpDir);
+  fs.mkdirSync('./unix-socket-dir');
+  cluster.fork().on('exit', common.mustCall(function(statusCode) {
+    assert.strictEqual(statusCode, 0);
+  }));
+} else {
+  process.chdir('./unix-socket-dir');
+  net.createServer(common.mustNotCall()).listen('./cluster.sock', function() {
+    assert.strictEqual(
+      fs.existsSync('./cluster.sock'), true,
+      'cluster.sock created in CWD');
+
+    process.exit(0);
```

Comment:
Instead of forcing the exit, can you cleanup as necessary and let the process exit naturally.